rotkehlchen/tests/utils/globaldb.py

```diff
@@ -107,3 +110,20 @@ def patch_for_globaldb_migrations(stack: ExitStack, new_list: list) -> ExitStack
         ),
     )
     return stack
+
+
+def globaldb_get_general_cache_last_queried_ts(
+        cursor: DBCursor,
+        key_parts: Iterable[Union[str, GeneralCacheType]],
+        value: str,
+) -> Optional[Timestamp]:
+    """Function to get timestamp at which pair key - value was queried last time."""
+    cache_key, _ = compute_cache_key(key_parts)
+    cursor.execute(
+        'SELECT last_queried_ts FROM general_cache WHERE key=? AND value=?',
+        (cache_key, value),
+    )
+    result = cursor.fetchone()
+    if result is None:
+        return None
```

Comment:
I did not build to function.

I was just recommended by Yabir to carry those from cache.py to a test util file since it is only used in tests.

But I guess since it is optional to Timestamp, it should probably remain as such!?